cves/kernel/CVE-2013-2058.yml

```diff
@@ -305,9 +183,9 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: "no record of discussions"
```

Comment:
You previously described a discussion which occurred with NXP folks.

Reply:
there wasn't really a discussion about the issue, just a developer at NXP finding it and recommending a fix, which they took.